src/toolkits/object_detection/od_data_iterator.cpp

```diff
@@ -304,6 +309,10 @@ simple_data_iterator::compute_properties(
     result.classes = std::move(expected_class_labels);
     int i = 0;
     for (const std::string& label : result.classes) {
+      if (classes_inferred.find(label) == classes_inferred.end()) {
+        std::cout << "Warning: class label '" + label +
```

Comment:
you can use a log stream with a warning flag.

Reply:
+1 to that. Please don't use count. log progress_stream is better.

Reply:
use `logprogress_stream << "Warning: class label '" ...`

logpgress_stream has a call back at the python layer!